update/emoncms.sh

```diff
@@ -271,7 +271,7 @@ echo "------------------------------------------"
 if [ ! -f /etc/systemd/system/mariadb.service.d/override.conf ]; then
   echo "Installing mariadb service config"
   sudo mkdir -p /etc/systemd/system/mariadb.service.d/
-  sudo cp /home/pi/emonpi/stretch/mariadb-service-d-override.conf /etc/systemd/system/mariadb.service.d/override.conf
+  printf "[Service]\nProtectHome=false\n" | sudo tee /etc/systemd/system/mariadb.service.d/override.conf
```

Comment:
I think any file in this folder is read as an override (not 100% sure). If so, would calling it `emoncms.conf` be better, then it could be specifically targetted for deletion if not required at a later stage.